web-console/knikubevirt/vm-list/vm-list.md

```diff
@@ -0,0 +1,85 @@
+# VM List
+
+![List of VMs](img/0-0-main.png)
+
+The VM list contains columns for the following:
+- Name
+- Namespace
+- Status
+  - Running, Off, Error (with custom messages), Cloning, Migrating, Importing, Warning, Pending (with changes that will apply after restart), Unknown
+- Created
+  - Date formatting should follow OpenShift's console, viewable [here](https://github.com/openshift/console/blob/master/frontend/public/components/utils/timestamp.jsx) and [here](https://github.com/openshift/console/blob/master/frontend/public/components/utils/datetime.ts)
+  - If possible, a tooltip should appear on hover with the exact creation date, including year.
+- Node
+- IP Address
+- FQDN
```

Comment:
This table looks very cramped for space. 6 columns seems to be the max number we will display in list views. Is there something you can drop? Also wondering if this is in priority order because responsive states will drop columns from right to left.